cmake/modules/FindIso9660pp.cmake

```diff
@@ -15,14 +15,16 @@ if(PKG_CONFIG_FOUND)
   pkg_check_modules(PC_ISO9660 libiso9660>=2.1.0 QUIET)
 endif()
 
+find_package(Cdio)
```

Comment:
```suggestion
find_package(Cdio REQUIRED)
```

Reply:
As `iso9660` is a separate optional feature and not required by many Kodi installations, it break ability to build Kodi without `iso9660` lib.